docs/website/docs/user-guides/advanced/connecting-a-service.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Connecting to a Service
+sidebar_position: 1
+---
+
+Building on top of the [Go quickstart guide](../quickstart/go.md), this guide will extend the application to connect it to a deployable MongoDB service.
+
+## Prerequisites
+1. [Install the Service Binding Operator via Operator Hub](https://operatorhub.io/operator/service-binding-operator). Read [here](../../command-reference/add-binding.md#description) to know why this is required.
+
+2. [Install Percona Server Mongodb Operator via Operator Hub](https://operatorhub.io/operator/percona-server-mongodb-operator).
+:::note
+The operator will be installed in a new namespace called "my-percona-server-mongodb-operator" and will be usable from that namespace only.
+:::
+3. Create a MongoDB service.
+
+import CreateMongodbService from './_create-mongodb-service.mdx';
+
+<CreateMongodbService/>
+
+## Step 0. Implement the code logic
```

Comment:
We put "Step 0" if it's an optional step :) This hould be step 1.

Reply:
I thought we were just following the programming indexing system where you begin with 0. Also, I still have to work on a few changes that you've requested, so please wait until I'm done with those before you review again.